opentelemetry-sdk/src/opentelemetry/sdk/context/contextvars_context.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from contextvars import ContextVar
+from sys import version_info
+
+from opentelemetry.context import Context
+from opentelemetry.context.context import RuntimeContext
+
+if (3, 5, 3) <= version_info < (3, 7):
+    import aiocontextvars  # type: ignore # pylint:disable=unused-import
+
+elif (3, 4) < version_info <= (3, 5, 2):
+    import opentelemetry.sdk.context.aiocontextvarsfix  # pylint:disable=unused-import
+
+
+_CONTEXT_KEY = "current_context"
```

Comment:
This can be moved to the parent class, or if not, at least in this module just put its value where it is needed instead of having a module variable.

Reply:
moved to the class instead